src/main/java/com/es/phoneshop/web/MiniCartServlet.java

```diff
@@ -0,0 +1,38 @@
+package com.es.phoneshop.web;
+
+import com.es.phoneshop.service.CartService;
+import com.es.phoneshop.service.impl.DefaultCartService;
+import com.es.phoneshop.model.cart.Cart;
+
+import javax.servlet.ServletConfig;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+public class MiniCartServlet extends HttpServlet {
+
+    private CartService cartService;
+
+    @Override
+    public void init(ServletConfig config) throws ServletException {
+        cartService = DefaultCartService.getInstance();
+    }
+
+    @Override
+    protected void doGet(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
+        requestLogic(request, response);
+    }
+
+    @Override
+    protected void doPost(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
+        requestLogic(request, response);
+    }
+
+    private void requestLogic(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
```

Comment:
Methods are actions, thus their names are always verbs. Rename it to smth like handleRequest or whatever more appropriate